runtime/browser/runtime.cc

```diff
@@ -60,6 +60,7 @@ Runtime::Runtime(content::WebContents* web_contents)
     : WebContentsObserver(web_contents),
       web_contents_(web_contents),
       fullscreen_options_(NO_FULLSCREEN),
+      remote_debugging_enabled_(false),
```

Comment:
we never use this data member, it should be just removed from the class declaration